engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -230,7 +227,18 @@
 public class NetworkOrchestrator extends ManagerBase implements NetworkOrchestrationService, Listener, Configurable {
     static final Logger s_logger = Logger.getLogger(NetworkOrchestrator.class);
 
-    private static final String KVM_MTU_SIZE = "kvm.mtu.size";
+    public static final ConfigKey<Integer> NetworkGcWait = new ConfigKey<Integer>(Integer.class,
+        "network.gc.wait", "Advanced", "600",
```

Comment:
Why are these here? I mean, they already exist in 4.13?

Reply:
network.gc.interval and network.gc.wait

Reply:
I did not add these keys, just moved them from the bottom up.